python/ray/serve/_private/proxy.py

```diff
@@ -758,7 +759,10 @@ async def send_request_to_replica(
                 message=message,
             )
         except Exception as e:
-            logger.exception(e)
+            if isinstance(e, (RayActorError, RayTaskError)):
```

Comment:
Is it possible to implement this behavior using an extra `except` block instead of a conditional?

Reply:
yes just adds more redundancy

Reply:
right now we have one `except` block per status code, wanted to keep that structure